control/veda_deliver.py

```diff
@@ -22,34 +28,21 @@
 from veda_utils import ErrorObject, Metadata, Output, VideoProto
 from veda_val import VALAPICall
 from veda_video_validation import Validation
-from watchdog import Watchdog
 
 try:
     from control.veda_deliver_3play import ThreePlayMediaClient
 except ImportError:
     from veda_deliver_3play import ThreePlayMediaClient
 
-
 LOGGER = logging.getLogger(__name__)
 
-
-
 try:
     boto.config.add_section('Boto')
 except:
     pass
 boto.config.set('Boto', 'http_socket_timeout', '100')
 
-
-"""
-VEDA Delivery class - determine the destination
-and upload to the appropriate endpoint via the approp. methods
-
-
-"""
 homedir = expanduser("~")
-
-
 watchdog_time = 10.0
```

Comment:
Do we need this, given that we've deleted watchdog file

Reply:
Ok